### PR TITLE
Update nf-ioapiset-getoverlappedresult.md

### PR DESCRIPTION
**Pull request by heroboy**

### CODE REVIEW

**Comment by PRMerger10** [comment]
@heroboy : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: sdk-api-src/content/ioapiset/nf-ioapiset-getoverlappedresult.md [hunk 1]
```diff
@@ -69,8 +70,9 @@ A handle to the file, named pipe, or communications device. This is the same han
 <a href="https://docs.microsoft.com/windows/desktop/api/fileapi/nf-fileapi-writefile">WriteFile</a>, 
 <a href="https://docs.microsoft.com/windows/desktop/api/namedpipeapi/nf-namedpipeapi-connectnamedpipe">ConnectNamedPipe</a>, 
 <a href="https://docs.microsoft.com/windows/desktop/api/namedpipeapi/nf-namedpipeapi-transactnamedpipe">TransactNamedPipe</a>, 
-<a href="https://docs.microsoft.com/windows/desktop/api/ioapiset/nf-ioapiset-deviceiocontrol">DeviceIoControl</a>, or 
-<a href="https://docs.microsoft.com/windows/desktop/api/winbase/nf-winbase-waitcommevent">WaitCommEvent</a> function.
+<a href="https://docs.microsoft.com/windows/desktop/api/ioapiset/nf-ioapiset-deviceiocontrol">DeviceIoControl</a>, 
+<a href="https://docs.microsoft.com/windows/desktop/api/winbase/nf-winbase-waitcommevent">WaitCommEvent</a>, or 
+<a href="https://docs.microsoft.com/windows/win32/api/fileapi/nf-fileapi-lockfileex">LockFileEx</a> function.
```
Comment (AjayVV) [inline sdk-api-src/content/ioapiset/nf-ioapiset-getoverlappedresult.md]: A call to `ReadDirectoryChangesW` is also a good candidate.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Comment by Karl-Bridge-Microsoft** [comment]
#sign-off

**Comment by PRMerger18** [comment]
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@Karl-Bridge-Microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Comment by Karl-Bridge-Microsoft** [comment]
#sign-off

**Pull request merged**